Find a functionally equivalent code:android.app.Activity.onWindowStartingActionMode:COMMENT
Method Modifier: public      
Comment:/**
 * Give the Activity a chance to control the UI for an action mode requested
 * by the system.
 *
 * <p>Note: If you are looking for a notification callback that an action mode
 * has been started for this activity, see {@link #onActionModeStarted(ActionMode)}.</p>
 *
 * @param callback The callback that should control the new action mode
 * @return The new action mode, or <code>null</code> if the activity does not want to
 * provide special handling for this action mode. (It will be handled by the system.)
 */

@@ -1,5 +1,5 @@
 {
-    initActionBar();
+    initWindowDecorActionBar();
     if (mActionBar != null) {
         return mActionBar.startActionMode(callback);
     }

